sim/battle.ts

```diff
@@ -2379,6 +2382,16 @@ export class Battle {
 						this.actions.switchIn(side.pokemon[i], i);
 					}
 				}
+				if (this.gameType === 'rotation') {
+					for (let i = 1; i < 3; i++) {
+						// temporarily flag as active for Illusion, etc
+						// TODO is this actually needed?
+						side.pokemon[i].isActive = true;
+						this.runEvent('BeforeSwitchIn', side.pokemon[i]);
+						this.add('switch', side.pokemon[i], side.pokemon[i].getDetails);
```

Comment:
Should this be using `this.actions.switchIn`?

Reply:
I avoid `switchIn` because effects such as Intimidate shouldn't activate for the 2nd and 3rd Pokemon.

Reply:
Ah, I was confused by `Ilusion, etc` - is this here only to set up Illusion? (When does the ability of the 2nd or 3rd Pokémon actually start?)

Reply:
They don't. This is the case for both gen 5 and 6.